From ff9d5e4273cd08ae739b6a132de2e1757fe40cd5 Mon Sep 17 00:00:00 2001
From: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Date: Fri, 31 Aug 2018 11:30:23 +0900
Subject: [PATCH 347/909] media: rcar-vin: Add memory type of VB_USERPTR
 support

commit 321dd11fbe901a417d0b08ed44a2a0b56459d2c7 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

The memory type of USERPTR is supported in rcar-vin driver.

Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/media/platform/rcar-vin/rcar-dma.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/rcar-vin/rcar-dma.c b/drivers/media/platform/rcar-vin/rcar-dma.c
index da2a1c0cfc61..3a783e6ba3f7 100644
--- a/drivers/media/platform/rcar-vin/rcar-dma.c
+++ b/drivers/media/platform/rcar-vin/rcar-dma.c
@@ -1369,7 +1369,7 @@ int rvin_dma_register(struct rvin_dev *vin, int irq)
 
 	/* buffer queue */
 	q->type = V4L2_BUF_TYPE_VIDEO_CAPTURE;
-	q->io_modes = VB2_MMAP | VB2_READ | VB2_DMABUF;
+	q->io_modes = VB2_MMAP | VB2_READ | VB2_DMABUF | VB2_USERPTR;
 	q->lock = &vin->lock;
 	q->drv_priv = vin;
 	q->buf_struct_size = sizeof(struct rvin_buffer);
-- 
2.17.1

